src/VirtoCommerce.Platform.Security/CustomPasswordValidator.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using VirtoCommerce.Platform.Core.Security;
+using VirtoCommerce.Platform.Security.Repositories;
+
+namespace VirtoCommerce.Platform.Security
+{
+    /// <summary>
+    /// Overriding https://github.com/dotnet/aspnetcore/blob/release/3.1/src/Identity/Extensions.Core/src/PasswordValidator.cs
+    /// </summary>
+    public class CustomPasswordValidator : PasswordValidator<ApplicationUser>
```

Comment:
Should we mark this class `sealed`?

Reply:
the base PasswordValidator is extendible. Why sealing?

Reply:
CustomPasswordValidator is already is a special case. If the user needs an additional password validator he can implement it via IPasswordValidator\<T\>. By the way, where is `CustomPasswordValidator` registration? Maybe be I miss it, but I cant find it.